sdk/ai/ai-projects/package.json

```diff
@@ -89,6 +89,10 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^4.3.0",
     "@azure/monitor-opentelemetry-exporter": "^1.0.0-beta.31",
+    "@azure/opentelemetry-instrumentation-azure-sdk": "^1.0.0-beta.7",
```

Comment:
hm it's a little odd to me that these are now needed if they were not needed before. Any idea why? I'm less concerned about devDependencies but mostly curious

Reply:
Cleaned up these dev deps.